DeAceroDLabs/marketplace-frontend
# Fix/dmp 535 Aligned delete and edit buttons

## What is done

[[DMP-535] Aligned delete and edit buttons](https://deacero.atlassian.net/browse/DMP-535)

## How is done

- Aligned edit and delete buttons from `ShoppingCartItem component.

## How to test

1. Run the project via `npm start`.
2. /carrito/productos.
3. You should see the edit and delete buttons correctly aligned.
